inference-engine/tests/ie_test_utils/functional_test_utils/layer_test_utils.cpp

```diff
@@ -148,6 +149,14 @@ std::vector<std::vector<std::uint8_t>> LayerTestsCommon::CalculateRefs() {
             // reference inference on device with other options and nGraph function has to be implemented here
             break;
         }
+        case INTERPRETER_TRANSFORMATIONS: {
+            auto cloned_function = ngraph::clone_function(*function);
+
+            // todo: add functionality to configure the necessary transformations
+            ngraph::pass::ConvertOpSet2ToOpSet1().run_on_function(cloned_function);
```

Comment:
Why do we need this pass here?

Reply:
The function `decompose ` of fused op used for validation BatchToSpace/SpaceToBatch on INTERPRETER backend, now this functionality was moved to transformation. It might be better to call the `ConvertBatchToSpace / SpaceToBatch` separately instead of `ConvertOpSet2ToOpSet1()`

Reply:
Calling ConvertBatchToSpace / SpaceToBatch directly is much better